src/Uno.UWP/VisualTreeHelperHelper.cs

```diff
@@ -0,0 +1,15 @@
+﻿#nullable enable
+
+using System;
+
+namespace Uno
+{
+	internal static class VisualTreeHelperHelper
```

Comment:
The duplication may feel weird. But two reasons:

1. Following the same naming done for `DispatcherTimerHelper` (ie, type name existing in one layer but not the otherr + "Helper"). Here, the type name in itself is "VisualTreeHelper".
2. Avoid name clashes between the two types

Reply:
Please move this type to `Helpers` folder. As for the naming, maybe `VisualTreeHelperProxy` would be nicer. We could also rename the other to `DispatcherTimerProxy` - but let's see if we could even use the extension facility instead, that would be even cleaner.

Reply:
@MartinZikmund I moved it to `Helpers`. (Note that `DispatcherTimeHelper` is in `Uno` folder not `Helpers` folder)

Reply:
While you are at it, you can move the `DispatcherTimeHelper` too, for cleanup purposes :-)